bee-api/bee-rest-api/src/types/dtos.rs

```diff
@@ -894,7 +894,8 @@ impl TryFrom<&NativeTokenDto> for NativeToken {
                 .0
                 .parse::<U256>()
                 .map_err(|_| Error::InvalidField("amount"))?,
-        ))
+        )
+        .map_err(|_| Error::InvalidField("NativeTokens"))?)
```

Comment:
ocd trigger alarm: shouldn't "amount" in line 896 also be uppercase?

Reply:
Also, wouldn't this and the token_id error never be returned because of `map_err(|_|`?

Reply:
No but actually it's NativeTokens that should be nativeTokens, I'll change

Reply:
> Also, wouldn't this and the token_id error never be returned because of map_err(|_|?

You are totally right and I told myself the same but that's how this whole crate is done, it definitely needs a refactoring